scripts/codemod/modifyProp.js

```diff
@@ -0,0 +1,96 @@
+// @flow strict
+
+/**
+ * CODEMOD to MODIFY (DEPRECATE or  RENAME) GESTALT COMPONENT PROPS
+ * Ex. <Box size="" /> to <Box />
+ * Ex. <Box size="" /> to <Box renamedSize="" />
+ * Ex. <Dropdown.Item size="" /> to <Dropdown.Item renamedSize="" />
+ *
+ *
+ * OPTIONS:
+ * --action: 'deprecate' removes prop matching previousPropName, 'rename' replaces prop name matching previousPropName with nextPropName
+ * --componentName: component to which modify props
+ * --subcomponentName: component's subcomponent to which modify props
+ * --previousPropName: current prop name to be replaced
+ * --nextPropName: (only required with rename action) new prop name to replace with
+ *
+ *
+ * TO RUN THIS CODEMOD
+ * yarn run:codemod modifyProp  <folder/file path> --action=<value> --previousPropName=<value> --nextPropName=<value>
+ * E.g. yarn run:codemod modifyProp  ~/code/pinboard/webapp --action=rename --componentName=Box --previousPropName=size --nextCmpName=renamedSize
+ * E.g. yarn run:codemod modifyProp  ~/code/pinboard/webapp --action=rename --componentName=Dropdown --subcomponentName=Item --previousPropName=size --nextCmpName=renamedSize
+ * E.g. yarn run:codemod modifyProp  ~/code/pinboard/webapp --action=deprecate   --componentName=Box --previousCmpName=size
+ */
+
+import {
+  getImports,
+  getJSX,
+  getLocalImportedName,
+  initialize,
+  isNotGestaltImport,
+  matchesComponentName,
+  getNewAttributes,
+  replaceJSXAttributes,
+  saveSource,
+  sourceHasChanges,
+  throwErrorIfSpreadProps,
+} from './helpers/codemodHelpers.js';
+
+// $FlowFixMe[unclear-type]
+type GenericType = any;
```

Comment:
The alias is meant to prevent having to disable // $FlowFixMe[unclear-type]
in every line. 

we flow-typed it to any here `flow-typed/npm/jscodeshift_vx.x.x.js`